src/Frontend/Page/Idea.hs

```diff
@@ -396,7 +387,7 @@ instance FormPage CommentIdea where
     redirectOf (CommentIdea idea _) = U.viewIdeaAtComment idea . view _Id
 
     makeForm CommentIdea{} =
-        "comment-text" .: (CommentContent <$> validateMarkdown "Verbesserungsvorschlag" (DF.string Nothing))
+        "comment-text" .: (CommentContent <$> validateMarkdown "Verbesserungsvorschlag" (Markdown <$> DF.text Nothing))
 
     -- FIXME styling
```

Comment:
I don't think these are very helpful either.  Styling issues are kept track of elsewhere, and having them here just risks that they will remain after styling has been fixed.